_pages/install/browser-support.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Web Browser Support
+---
+
+Ideally, your storefront will render uniformly in any browser that is used by your customers. However, web browsers do not all support the same web standards, nor do they support the same standards in the same way. For example, there are different ways of interpreting a standard's specifications, deliberate design decisions that result in different outcomes, as well as bugs that are unique to each browser. And for each browser that is supported, there is a resulting implementation effort that includes QA, architectural decisions, and technical decisions. In the end, a balance has to be struck between providing an excellent storefront experience, and supporting (as much as possible) the different web browsers that are out there.
```

Comment:
> nor do they support the same standards in the same way

nor do they interpreted the standards in the same way

(avoiding same twice)

Reply:
In this case, the repetition was intentional, but I like your suggestion and I'm happy to make the change 😄

Reply:
Actually, speaking of repetition 🙂 , the word "interpret" is used at the beginning of the very next sentence, so I prefer not to use it here. I propose "nor do they _handle_ these standards in the same way"

Would that work?